main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	l "FDJ_SLACK/log"
+	"FDJ_SLACK/parser"
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+var win = []int{7, 14, 22, 28, 42}
+var luckWin = 5
+
+var slackURL = os.Getenv("SLACK_HOOK_URL")
+var localhost = "http://localhost:8888"
+
+func main() {
+	if slackURL == "" {
+		slackURL = localhost
+	}
+
+	p := parser.NewParser()
+	data := p.FetchData()
+	result := p.ParseData(data)
+	for index, result := range result {
+		if index == 0 { // only first result
+			l.Info(result)
+			postToSlack(slackURL, result.String())
```

Comment:
Do not pass `slackURL` as a parameter to the postToSlack method but use it directly instead ?

Reply:
Yes, this is what I do at the beggining of the main

if slackURL == "" {
	slackURL = localhost
}